resources/tests/org.eclipse.core.tests.resources.saveparticipant/src/org/eclipse/core/tests/resources/saveparticipant/SaveManager2Test.java

```diff
@@ -76,16 +55,16 @@ public void testBuilder() throws CoreException {
 		verifier.addExpectedChange(added, project, IResourceDelta.ADDED, 0);
 		added.create(createRandomContentsStream(), true, null);
 		waitForBuild();
-		assertTrue("3.2", verifier.wasAutoBuild());
-		assertTrue("3.3", verifier.isDeltaValid());
+		assertTrue(verifier.wasAutoBuild());
+		assertTrue(verifier.isDeltaValid());
 		// remove the file because we don't want it to affect any other delta in the test
 		added.delete(true, false, null);
 	}
 
 	public void testSaveParticipant() throws Exception {
 		// get plugin
 		Bundle bundle = Platform.getBundle(PI_SAVE_PARTICIPANT_1);
-		assertTrue("0.1", bundle != null);
+		assertTrue(bundle != null);
```

Comment:
```suggestion
		assertNotNull(bundle);
```

Reply:
Thank you! I missed that one. Fixed in 539406b.